### DISCUSSION TITLE
챈 잘생겼다

### DISCUSSION THREAD

**Discussion by moolmin** [100-hours-a-week/discussions#1] [General]
![-3-2-2](https://github.com/user-attachments/assets/93065e01-2ee6-44e1-9f06-4254fae9b9f7)

![PhotoFunia-1723792827](https://github.com/user-attachments/assets/f3aebbf7-57dd-45ba-b874-9df18fc2464d)

**Comment by startupcodekr** [comment]
첸 잘 생겼다

**Comment by Ssun2zang** [comment]
첸 부럽다
첸 귀엽다
첸 착하다
첸 멋지다
첸 섹시하다
첸 청순하다
첸 예쁘다
첸 잘생겼다

"첸"

**Reply by Namgyu11** [reply]
헤이즐리 NO.1

**Reply by judy-kimsohui** [reply]
와사귀

**Comment by junseoparkk** [comment]
아무에게도 뺏기지 않을거에요

**Comment by judy-kimsohui** [comment]
첸 내꺼예요
